### PR TITLE
Feature(copy tool): add copy tool2 command

### PR DESCRIPTION
**Pull request by 0fatal**
#325

### CODE REVIEW

File: internal/task/task/install/install_tool.go [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+
+	if err = checkPathExist(path, hc.GetSSHConfig(), curveadm); err != nil {
+		return nil, err
+	}
+	if err = checkPathExist(confPath, hc.GetSSHConfig(), curveadm); err != nil {
+		return nil, err
+	}
+
+	subname := fmt.Sprintf("host=%s", dc.GetHost())
+	t := task.NewTask("Install tool v2", subname, hc.GetSSHConfig())
```
Comment (caoxianfei1) [inline internal/task/task/install/install_tool.go]: suggestion：
Copy version 2 tool to host

File: cli/command/install/cmd.go [hunk 2]
```diff
@@ -0,0 +1,21 @@
+package install
```
Comment (caoxianfei1) [inline cli/command/install/cmd.go]: Add copyright.

File: cli/command/install/tool.go [hunk 3]
```diff
@@ -0,0 +1,89 @@
+package install
```
Comment (caoxianfei1) [inline cli/command/install/tool.go]: Ditto.

File: internal/task/task/install/install_tool.go [hunk 4]
```diff
@@ -0,0 +1,79 @@
+package install
```
Comment (caoxianfei1) [inline internal/task/task/install/install_tool.go]: Ditto.

File: internal/tui/common/prompt.go [hunk 5]
```diff
@@ -71,6 +71,9 @@ to watch the formatting progress.
 `
 	PROMPT_CANCEL_OPERATION = `[x] {{.operation}} canceled`
 
+	PROMPT_PATH_EXIST = `{{.path}} already exists.
+`
```
Comment (caoxianfei1) [inline internal/tui/common/prompt.go:75]: redundant one line?

**Review by caoxianfei1 — APPROVED** [review]
LGTM!

**Comment by caoxianfei1** [comment]
@Wine93 PTAL~

**Review by Wine93 — APPROVED** [review]

**Pull request merged**